deploy/ray-operator/config/samples/ray_v1_raycluster.bk.yaml

```diff
@@ -0,0 +1,294 @@
+apiVersion: ray.io/v1
+kind: RayCluster
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: raycluster-sample
+spec:
+  clusterName: raycluster-sample
+  images:
+    defaultImage: "reg.docker.alibaba-inc.com/volvo_ci_arcx/ray_package:a59ce20ec8b9876a67df84ca45dc456481bc2d3a"
+    clusterApiServerImage: "reg.docker.alibaba-inc.com/zeromem/clusterapiserver:20191008"
+  imagePullPolicy: "Always"
+
+  # optional `extensions`, and all sub-options are optional.
+  extensions:
+    - size: 2
+      groupName: group1
+      type: worker
+      idList: ["raycluster-sample-group1-worker-0","raycluster-sample-group1-worker-1"]
+
```

Comment:
The below properties “lable“,”affinity” etc.  are podTemplate

Reply:
this file deleted.This is the CR example , allows user to set this props

Reply:
To simplify CRD, i think we can use podTemplate instead of pod spec . Refer to deployment,statefulset.

Reply:
In CRD, we didn't use podTemplate directly, there are some reasons
- users needn't know all the configurations in Kubernetes, this will add complexity to user
- podTemplate is already an available Kubernetes attribute, it'll be hard to do some logic check or field check
- In operator there is no such practice that uses podTemplate directly for now, the reason we use CRD is that we control the final